CGPs/0015.md

```diff
@@ -0,0 +1,39 @@
+# CGP [0015]: Extend Attestation Expiration Duration
+
+- Date: 2020-11-13
+- Author(s): @aslawson
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
```

Comment:
For reviewing this proposal, we can make another item called `Discussions Link` and add a forum post link there. This will allow folks to know where to discuss this topic itself.

Reply:
@aslawson can you add `Discussions Link` with a link to the forum post for discussions? You just need to make a forum post for this CGP. Otherwise it's good to go for me to approve this Draft.

Reply:
sure -- can you redirect me to that document you had that has the steps involving CGP process?

Reply:
You're right that we don't have anything specified, I'm just using the CIP process as a reference for this. It's generally good practice to add a discussions link for a spec document so folks can track all relevant discussions on it. I went ahead and created a forum post for discussions and linked it to your document. Gonna approve and merge your PR now.